server/server/data_loader/po.py

```diff
@@ -87,12 +88,19 @@ def process_dir(change_tracker, po_dir, info):
         data = extract_strings_from_po_file(po_file)
         uid = remove_leading_zeros(po_file.stem)
 
+        root_author = get_author(info['root_author'], authors)
```

Comment:
You can merge `get_author` and `get_author_short` functions into one and return tuple. eg:
`root_author, root_author_short = get_author_names(info['root_author'], authors)`
This way you don't have to iterate over authors list twice.

Reply:
Thanks. Done that also for some older code.